### PR TITLE
Random decay time

### LINKED ISSUE #3148: Magic Wall and Wild Growth missing changes
- [x] magic wall should have a random duration between 14 and 20 seconds when shot
- [x] wild growth should have a random duration between 38 and 45 seconds when shot
- [x] ITEM_WILDGROWTH_NOPVP and ITEM_MAGICWALL_NOPVP should be destroyed on stepIn by any player (only on NOPVP world types or NOPVP tiles)

[since 10.01 update](https://tibia.fandom.com/wiki/Updates/10.01)

### PR DESCRIPTION
**Pull request by EPuncker**
Co-Authored-By: Sarah Wesker <sarahelizabetwesker@gmail.com>

### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
Added the possibility to items to have random decay time

**Issues addressed:** 
closes #3148

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: data/items/items.xml [hunk 1]
```diff
@@ -706,15 +706,17 @@
 	<item id="1497" article="a" name="magic wall">
 		<attribute key="type" value="magicfield" />
 		<attribute key="decayTo" value="0" />
-		<attribute key="duration" value="20" />
+		<attribute key="durationMin" value="14" />
+		<attribute key="durationMax" value="20" />
```
Comment (ranisalt) [inline data/items/items.xml:710]: What about `<attribute key="duration" minvalue="14" maxvalue="20" />`, or even `min` and `max`?

Comment (MillhioreBT) [reply data/items/items.xml:710]: It doesn't seem like a bad idea, I like the way you think

Comment (EPuncker) [reply data/items/items.xml:710]: but how would it work with the remaining items that use duration? 3 parameters?

Comment (ranisalt) [reply data/items/items.xml:710]: It's either `value`, or `min` and `max`. If the item specifies `value`, don't even try to read the others.

Comment (yamaken93) [reply data/items/items.xml:710]: Maybe value, minValue and maxValue. Somehow i don't like how its called value then it just min or max.

Comment (Zbizu) [reply data/items/items.xml:710]: I suggest duration and durationMax - would be consistent with count and countMax in monsters loot

**Review by DSpeichert — COMMENTED** [review]
I'm technically in favor of proper XML, copying @ranisalt's comment:
> It's either value, or min and max. If the item specifies value, don't even try to read the others.

**Comment by EPuncker** [comment]
closed because no interest from anyone in helping

**Pull request closed without merging**